### PR TITLE
Enhancement-View-Charge

### PR DESCRIPTION
**Pull request by rrpawar96**
## Description
Describe the changes made and why they were made instead of how they were made.

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [ ] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by rrpawar96** [comment]
ViewChargeEnhancement

**Comment by rrpawar96** [comment]
@BLasan @luckyman20 Please review this view charge enhancement PR

File: app/scripts/controllers/product/ViewChargeController.js [hunk 1]
```diff
@@ -5,6 +5,22 @@
...
             resourceFactory.chargeResource.get({chargeId: routeParams.id}, function (data) {
                 scope.charge = data;
+
+                resourceFactory.loanProductResource.get({resourceType: 'template'}, function (data) {
+                    scope.product = data;
+                    scope.paymentTypeOptions = data.paymentTypeOptions;
+                    const i = 1;
+                    scope.filteredItems = scope.product.repaymentFrequencyTypeOptions.slice(0, i).concat(scope.product.repaymentFrequencyTypeOptions.slice(i + 1, scope.product.repaymentFrequencyTypeOptions.length));
+
+                    var x = scope.charge.restartFrequencyEnum;
```
Comment (luckyman20) [inline app/scripts/controllers/product/ViewChargeController.js]: Avoid using variables like `i` and `x`. Please use more descriptive variables.

File: app/scripts/controllers/product/ViewChargeController.js [hunk 2]
```diff
@@ -5,6 +5,22 @@
...
+                    scope.restartEnum = scope.charge.restartFrequencyEnum;
+
+                    angular.forEach(scope.filteredItems, function(value, key){
```
Comment (luckyman20) [inline app/scripts/controllers/product/ViewChargeController.js]:
```suggestion
                    angular.forEach(scope.filteredItems, function(value, key) {
```

File: app/scripts/controllers/product/ViewChargeController.js [hunk 3]
```diff
@@ -5,6 +5,22 @@
...
+                        if(value.id == x){
```
Comment (luckyman20) [inline app/scripts/controllers/product/ViewChargeController.js]:
```suggestion
                        if(value.id == x) {
```

**Review by luckyman20 — COMMENTED** [review]
Few changes, else looks good.

**Comment by rrpawar96** [comment]
Thanks, @luckyman20 for reviewing the PR.
@bharathcgowda  Comment has been addressed.

**Comment by bharathcgowda** [comment]
@luckyman20  can you please confirm on the changes done are good for merging?

**Review by luckyman20 — APPROVED** [review]
LGTM!

**Comment by bharathcgowda** [comment]
thanks for the review @luckyman20

**Pull request merged**